torchinductor/scheduler.py

```diff
@@ -458,6 +559,13 @@ def __init__(self, nodes):
                 assert False, node
         self.name_to_node = {node.get_name(): node for node in self.nodes}
 
+        if bool(os.environ.get("INDUCTOR_DEBUG", False)):
+            global graph_dump_index
```

Comment:
So... we only create a single scheduler per graph we pass to inductor. I'd propose instead that we make a global variable that anybody can access for the purposes of logging.

Reply:
changed, we also need the PR here: https://github.com/pytorch/pytorch/pull/82368